superset-frontend/src/SqlLab/components/QueryAutoRefresh/index.tsx

```diff
@@ -97,6 +112,17 @@ function QueryAutoRefresh({
                   {},
                 ) ?? {};
               dispatch(refreshQueries(queries));
+              jsonPayload.result.forEach(query => {
```

Comment:
ok with disregarding @korbit-ai's comment given it's probably rare that we have multiple queries in a failed state, can probably afford the multi-rerender in those rare cases.